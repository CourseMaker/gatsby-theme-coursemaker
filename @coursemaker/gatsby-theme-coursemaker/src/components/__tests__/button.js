import React from "react"
import renderer from "react-test-renderer"
import Button from "../button"

describe("Button", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<Button />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})