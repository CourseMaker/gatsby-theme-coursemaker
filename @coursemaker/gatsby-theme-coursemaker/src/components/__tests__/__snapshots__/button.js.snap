// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<a
  className="btn btn-lg css-1v6n45o-Button"
  href="/"
/>
`;
